azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeRemoteParticipantLeftEventExtension.kt

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Please re-use existing `CallCompositeExtentions`. I think we should not create so many small files to put extentions in.